cscope.py

```diff
@@ -216,18 +216,15 @@ def match_output_line(self, line, mode):
         return output
 
     def run_cscope(self, mode, word):
-        newline = ''
-        if self.platform == "windows":
-            newline = '\r\n'
-        else:
-            newline = '\n'
+        newline = '\n'
 
         cscope_arg_list = [self.executable, '-dL', '-f', self.database, '-' + str(mode) + word]
         popen_arg_list = {
             "shell": False,
             "stdout": subprocess.PIPE,
             "stderr": subprocess.PIPE,
-            "cwd": self.root
+            "cwd": self.root,
+            "universal_newlines": True
```

Comment:
It looks like this is only important for Windows hosts. Could you move this to the ```if (self.platform == "windows")``` block below so we can make sure we don't cause regressions in non-Windows hosts?